pkgs/tools/system/swaysettings/default.nix

```
@@ -0,0 +1,75 @@
+{
+  pkgs,
+  meson,
```

Commas come before the arguments, nor after them.

```suggestion
  , meson
```

---

Ok corrected